change/@ni-nimble-components-612b4007-c1f6-4b64-92c7-3fb0494c1002.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is it just my eyes or do these colors look really similar? 😆

I'm not sure what the issue is; the theme-aware tokens look normal and the base colors in nimble-tokens look fine too. But the incorrect colors show up in all themes locally and in the Chromatic diff.

https://60e89457a987cf003efc0a5b-xuuatvbxpn.chromatic.com/?path=/docs/tokens-theme-aware-tokens--docs

<img width="637" alt="image" src="https://github.com/ni/nimble/assets/10500124/e1b69cf9-72e8-4213-a6f3-4be4849db5c4">

Reply:
Whoops. Copy-paste error.